server.js

```diff
@@ -0,0 +1,12 @@
+const dotenv = require('dotenv');
+dotenv.config();
+const port = process.env.PORT
+
+const express = require("express");
+const server = express();
+
+server.listen(port);
+
+server.use('/health', (req, res) => {
+	res.json({ status: 'UP' });
```

Comment:
Indenta usando 2 espacios, casi todos los linters sugieren eso.
```suggestion
  res.json({ status: 'UP' });
```